src/test/java/HmWrk19.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HmWrk19 extends BaseTest{
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        // Create a test case deletedPlaylist() using@ TEst annotations
+        // Use the reusable methods
+        // Use a @Parameter for passing baseUrl
+        //Given: navigate to "https://bbb.testpro.io/"
+        //Step 1: Log in with your credentials
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.